tRecorderApi/api/models/chapter.py

```diff
@@ -17,3 +21,127 @@ class Meta:
 
     def __str__(self):
         return '{}'.format(self.number)
+
+    @staticmethod
+    def getChaptersByProject(data):

```

Comment:
Lets try to remove this method, or heavily refactor it, before merging this in. Chapters should be able to be retrieved just from filtering, most of these python loops are unnecessary and inefficient. I thought this refactor was already done somewhere though?